devsetup/README.md

```diff
@@ -131,6 +131,40 @@ In case additional compute node VMs are deployed, run:
 make edpm_compute_cleanup EDPM_COMPUTE_SUFFIX=1
 ```
 
+### EDPM virtual baremetal deployment
```

Comment:
It would be goo to have a similar header at L94 above?

```
### EDPM deployment
#### EDPM virtual machine deployment
#### EDPM virtual baremetal deployment
```

Reply:
Good idea, I'll roll this into one of the follow-up PRs